### ISSUE TITLE
Implement basic BE CI

### ISSUE THREAD

**Issue by Corgam** [amosproj/amos2024ss04-building-information-enhancer#49]
## User story
1. As an AMOS course participant and an SD, I need to integrate basic CI into the BE, including formatting checks, lining checks, and unit tests. The tests do not have to be fully written yet, just put the infrastructure in place.
2. I want/need a separate npm command for each of the above-mentioned actions.
3. So that Dev-ops dev. can integrate it into the project's CI/CD. 

## Acceptance criteria
* Infrastructure done for the following:
  * Formatting checks
  * Linting checks
  * Unit tests
* Separate npm commands to run each of them

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
